package.json

```diff
@@ -21,6 +21,7 @@
     "js-yaml": "^3.4.6",
     "logops": "^1.0.0-alpha.3",
     "optimist": "^0.6.1",
+    "regex": "^0.1.1",
```

Comment:
I do not see a require(regex) in the previous update code, it was something missing previously?